client/source/js/modules/model/cost-coverage.html

```diff
@@ -70,8 +67,8 @@
               </p>
               <p>
                 3. [Optional] Specify scale-up parameters:
-                <input type="number" name="scaleUpParameter" min="1" max="10" step="0.1" class="txbox __inline __xl" placeholder="e.g.{{defaultScaleUpParameter}}"  ng-model="scaleUpParameter" ng-change="updateCurves()">
-                <error-messages for="scaleUpParameter" rules="{min:1, max:10}"></error-messages>
```

Comment:
@Ashalynd It does kind of look flat for OST with those parameters... but that's ok because it just means the user will need to pick different values. But what about for MSM programs with those same parameters? That doesn't look flat at all - not when I try it, anyway. How does it look to you?